sled-agent/src/bootstrap/server.rs

```diff
@@ -155,6 +156,9 @@ pub enum StartError {
 
     #[error("Failed to initialize lrtq node as learner: {0}")]
     FailedLearnerInit(bootstore::NodeRequestError),
+
+    #[error("Measurment error: {0}")]
```

Comment:
Following the error guide, we shouldn't display the inner value here with `{0}` and should instead mark it as `#[source]`. We have the same problem above, which is my fault!